### PR TITLE
CI examples update

### PR DESCRIPTION
**Pull request by mloubout**
Small update to CI due to growth and new contribution. All `example_*, *_example` are scanned for tesgt and all run so that yaml doesn't need update for new example (unless want to add example run).

### CODE REVIEW

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -60,13 +64,8 @@ jobs:
...
       run: |
-        py.test examples/seismic/elastic/elastic_example.py
-        py.test examples/seismic/viscoelastic/viscoelastic_example.py
-
-    - name: Seismic viscoacoustic examples
-      shell: bash -l {0}
-      run: |
-        py.test examples/seismic/viscoacoustic/viscoacoustic_example.py
+        python examples/seismic/elastic/elastic_example.py
+        python examples/seismic/viscoelastic/viscoelastic_example.py
```
Comment (georgebisbas) [inline .github/workflows/examples.yml:68]: This is a big expensive to execute compared to py.test.

Comment (mloubout) [reply .github/workflows/examples.yml:68]: This CI runs in ~8min, still way faster than pytest so doesn't really matter

Comment (georgebisbas) [reply .github/workflows/examples.yml:68]: ok, then. I am fine anyway.

Comment (FabioLuporini) [reply .github/workflows/examples.yml:68]: are errors caught as expected even w/o pytest ? 

pytest was gonna run `test_...` which has asserts inside

File: .github/workflows/examples.yml [hunk 2]
```diff
@@ -60,13 +64,8 @@ jobs:
     - name: Seismic elastic examples
       shell: bash -l {0}
       run: |
-        py.test examples/seismic/elastic/elastic_example.py
-        py.test examples/seismic/viscoelastic/viscoelastic_example.py
-
-    - name: Seismic viscoacoustic examples
-      shell: bash -l {0}
-      run: |
-        py.test examples/seismic/viscoacoustic/viscoacoustic_example.py
```
Comment (FabioLuporini) [inline .github/workflows/examples.yml:69]: Dropping because... ?

File: .github/workflows/examples.yml (hunk 1 above)
Comment (mloubout) [reply .github/workflows/examples.yml:68]: All tests are run above......

File: .github/workflows/examples.yml (hunk 2 above)
Comment (mloubout) [reply .github/workflows/examples.yml:69]: All tests are run above......

File: .github/workflows/examples.yml (hunk 1 above)
Comment (FabioLuporini) [reply .github/workflows/examples.yml:68]: if nothing can fail, why are we running these? for segfaults?

File: .github/workflows/examples.yml (hunk 2 above)
Comment (FabioLuporini) [reply .github/workflows/examples.yml:69]: right, I guess I'm an idiot again. So the py.test command searches for tests recursively in all subdirectories in all files named `*_example.py example_*.py`?

Comment (mloubout) [reply .github/workflows/examples.yml:69]: yep

File: .github/workflows/examples.yml (hunk 1 above)
Comment (mloubout) [reply .github/workflows/examples.yml:68]: Yeah these are 3D test is for 2D so making sure 3D doesn't error

**Pull request merged**